app/src/main/java/uk/ac/ebi/atlas/experimentpage/ExpressionAtlasContentService.java

```diff
@@ -174,73 +131,83 @@ public ImmutableList<ExternallyAvailableContent> list(String experimentAccession
                                                  String accessKey,
                                                  ExternallyAvailableContent.ContentType contentType) {
         Experiment<?> experiment = experimentTrader.getExperiment(experimentAccession, accessKey);
-        String externalResourceType = externalResourceLinksPriority(experiment);
         ImmutableList.Builder<ExternallyAvailableContent> arrayExpressAndOtherExternalResourcesLinks = ImmutableList.builder();
```

Comment:
We can just name this as `externalResourcesLinks`.

```suggestion
        var externalResourcesLinks = ImmutableList.builder();
```